src/App.tsx

```diff
@@ -1,61 +1,119 @@
+import React, { useState, useEffect } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import usersFromServer from './api/users';
```

Comment:
Directly importing `usersFromServer` here violates the requirement to prepare data in the `App` component and pass it down. Consider refactoring to fetch and prepare this data within the `App` component and pass it as props to child components.